packages/dynamoose/lib/Schema.ts

```diff
@@ -774,15 +774,15 @@ interface AttributeDefinition {
 export interface SchemaDefinition {
 	[attribute: string]: AttributeType | AttributeType[] | AttributeDefinition | AttributeDefinition[];
 }
-interface SchemaGetAttributeTypeSettings {
+export interface SchemaGetAttributeTypeSettings {
 	unknownAttributeAllowed: boolean;
 }
-interface SchemaGetAttributeSettingValue {
+export interface SchemaGetAttributeSettingValue {
 	returnFunction: boolean;
 	typeIndexOptionMap?: any;
 }
 
-interface SchemaInternalProperties {
+export interface SchemaInternalProperties {
```

Comment:
Why should something named `SchemaInternalProperties` be exported? That makes **ZERO** sense to me. Why are you needing this to be exported?

Reply:
Yeah that's fair, I can remove this export.